automation/test-linux.sh

```diff
@@ -49,7 +51,19 @@ EOF
 timeout=600
 sample=10
 
-oc wait --for=condition=Ready --timeout=${timeout}s dv/${TARGET}-datavolume-original -n $namespace
+oc wait --for=condition=Ready --timeout=${timeout}s dv/${dv_name} -n $namespace
+
+oc apply -n $namespace -f - <<EOF
+apiVersion: cdi.kubevirt.io/v1beta1
```

Comment:
isn't DataSource automatically created when you create VM from template?

Reply:
I think not, but I haven't tried it.
Where is the code or component that would create the DataSoruce ?

Reply:
And it would not make sense to create the `DataSource` when the **VM** is created. The `DataSource` has to exist and point to the latest golden image PVC when VM is created.